README.md

```diff
@@ -65,48 +73,7 @@ This project is built on __Raspberry Pi 3B+__ with __Raspbian OS__ and __Python
 
 <hr>
 
-- ### 4. Insert your IP on the app
```

Comment:
Note that this step is no longer necessary even outside docker. Please check my other code changes.